security-whitepaper/secure-defaults-cloud-native-8.md

```diff
@@ -0,0 +1,163 @@
+# **Secure Defaults: Cloud Native 8**
```

Comment:
nits: the headers don't need the bold, it should be there by default

Reply:
Good call out. The docs to markdown plugin added it and I missed it! Fixed it now.